src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -157,12 +157,11 @@ private IPCAWrapper BuildPCAWrapper(Guid clientId, Guid tenantId)
                     Identity.Client.LogLevel.Verbose,
                     enablePiiLogging: false,
                     enableDefaultPlatformLogging: true)
-                .WithWindowsBrokerOptions(new WindowsBrokerOptions
+                .WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows)

```

Comment:
Same here, a summary of why some of these things were changed would be super helpful. Thanks.

Reply:
Answered above